src/Drupal/DrupalExtension/Context/MarkupContext.php

```diff
@@ -107,11 +107,10 @@ public function assertRegionElementAttribute($tag, $attribute, $value, $region)
       foreach ($elements as $element) {
         $attr = $element->getAttribute($attribute);
         if (!empty($attr)) {
-          $found = TRUE;
-          if (strpos($attr, "$value") === FALSE) {
-            throw new \Exception(sprintf('The "%s" attribute does not equal "%s" on the element "%s" in the "%s" region on the page %s', $attribute, $value, $tag, $region, $this->getSession()->getCurrentUrl()));
```

Comment:
With this change though, if the attribute is found, but the value doesn't match, no exception is thrown below. A test to illustrate the current failed behavior would be useful I think.